app/models/effects/retire_subject.rb

```diff
@@ -9,7 +12,9 @@ def valid?
     end
 
     def self.config_fields
-      [:reason].freeze
+      # use 2-element response (key, choices) to enable dropdown in
+      # subject_rule_effect _form view, or 1-element (key) for text input
+      [[:reason, CONFIG_CHOICES]].freeze
```

Comment:
minor: I'd just inline these
```suggestion
      [[:reason, %w(classification_count flagged nothing_here blank consensus other human)]].freeze
```

Reply:
My logic here was to follow the pattern used by filters for validation (e.g., see [here](https://github.com/zooniverse/caesar/blob/master/app/models/filters/filter_by_training_behavior.rb)), as I think that would be a relatively straightforward enhancement in the future.  In that case, we'd end up using the list of acceptables again, so I'll keep the code as is.